resources/js/main.js

```diff
@@ -123,14 +123,10 @@ function addRow() {
   };
 
   // Add a new row to each page at incrementing index (see below)
-  for(i = 1; i <= numOfPages; i++) {
-    index = (pageSize * i) + t; // to insert at same relative pos. for each page
-    addRowAtIndex(index, newRow);
-    t += 1; // increment at each page to account for the new row
-  }
+  index = i ; // to insert at same relative pos. for each page
```

Comment:
What is i?

Reply:
It just works for some reason..im not sure why